### ISSUE TITLE
Post deployment trigger

### ISSUE THREAD

**Issue by medkbadri** [fluxcd/notification-controller#13]
Hello,

First, I want to thank you for your contribution to the GitOps Ecosystem.
Just want to know, if we can use this operator in order to trigger post-deployment action.

For example, once a deployment is achieved by Flux, I want another deployment to take place.

Thanks in advance.

**Comment by stefanprodan** [comment]
> For example, once a deployment is achieved by Flux, I want another deployment to take place.

Yes, this is covered by https://github.com/fluxcd/kustomize-controller/blob/master/docs/spec/v1alpha1/kustomization.md#kustomization-dependencies

**Comment by medkbadri** [comment]
Thank you, Stefan, for getting back to me
I started experimenting with the new gitops toolkit and I must admit that it's really cool!
I'm targeting the following use case:
- Deployment A: a new rollout update
- Workload B (Example: Tekton pipeline that will update an API GW once the Deployment A is rolled out )  (K8S manifests unchanged in Git)

I'm afraid customization dependencies won't suit this use case.

Side question: is it on your radar to implement deployment sync based on image tags like the old Flux

Thanks

**Comment by stefanprodan** [comment]
> is it on your radar to implement deployment sync based on image tags like the old Flux

Yes, the image update controller is listed on our roadmap here https://toolkit.fluxcd.io/roadmap/#flux-image-update-feature-parity
